### PR TITLE
new imaginary friend outfit

### PR DESCRIPTION
**Pull request by Mechanelize**
## About The Pull Request

added an imaginary friend staff officer outfit

## Why It's Good For The Game

xenos can have a xeno IF now marines can have a marine IF

## Changelog
:cl:
add: Added a new marine outfit for Imaginary Friends
/:cl:

### CODE REVIEW

**Comment by novaepee** [comment]
when the imaginary friend is a drill instructor

image

**Comment by goose4429** [comment]
Yes

**Comment by Mechanelize** [comment]
> when the imaginary friend is a drill instructor
> 
> 
> 
> image
> 
> 

Good idea

File: code/datums/jobs/job/other.dm [hunk 1]
```diff
@@ -89,5 +89,16 @@
...
+/datum/job/spatial_agent/marine_officer
+	outfit = /datum/outfit/job/other/spatial_agent/marine_officer
+
+/datum/outfit/job/other/spatial_agent/marine_officer
+	w_uniform = /obj/item/clothing/under/marine/officer/bridge
+	head = /obj/item/clothing/head/beret/marine
+	belt = /obj/item/storage/belt/gun/pistol/m4a3/officer
+	back = null
+	shoes = /obj/item/clothing/shoes/marine/full
+	wear_suit = /obj/item/clothing/suit/dummy/Xenonauten
```
Comment (Wayland-Smithy) [inline code/datums/jobs/job/other.dm:101]: CI/CD is crying about this line
`code/datums/jobs/job/other.dm:101:error: /obj/item/clothing/suit/dummy/Xenonauten: undefined type path`

Comment (Mechanelize) [reply code/datums/jobs/job/other.dm:101]: That shouldn't be there

**Pull request merged**